components/vocabularies/chebi/api/src/main/java/org/phenotips/vocabulary/internal/solr/ChEBIOntology.java

```diff
@@ -77,4 +77,12 @@ public String getName()
         result.add("ChEBI");
         return result;
     }
+
+    /**
+     * The url for the Chemical Entities of Biological Interest ontology website.
+     * @return a String representation of the url for the HPO website.
+     */
+    public String getUrlStr() {
```

Comment:
Add this method to the `Vocabulary` interface.

Better name: `getWebsite`.

Instead of getting the citation as a translation, add another method in `Vocabulary`: `getCitation`.